Makefile

```diff
@@ -30,6 +30,10 @@ all: test
 setup-dev: ## Initialise simulation tree with git hooks
 	@ln -s $(shell pwd)/setup/hooks/pre-commit $(shell pwd)/.git/hooks/pre-commit
 
+.PHONY: dep
```

Comment:
Is this purposely called "dep" or is it a typo for "dev"?

Reply:
Shouldn't it be "devtools"?

Reply:
copy and paste error! :) thanks